python/fledge/services/core/api/browser.py

```diff
@@ -149,7 +152,14 @@ async def asset(request):
         # Add the order by and limit, offset clause
         _and_where = prepare_limit_skip_payload(request, _where)
 
-    payload = PayloadBuilder(_and_where).ORDER_BY(["user_ts", "desc"]).payload()
+    # check the order. keep the default order desc
+    _order = 'desc'
+    if 'order' in request.query:
+        _order = request.query['order']
+        if _order not in ('asc', 'desc'):
+            raise web.HTTPBadRequest(reason="order must be asc or desc")
```

Comment:
Add message in body JSON as well.
`body=json.dumps({"message": "order must be asc or desc"})`